src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -440,7 +440,7 @@ private void FinishRequest(EasyRequest completedOperation, int messageResult)
                         completedOperation.EnsureResponseMessagePublished();
                         break;
                     default:
-                        completedOperation.FailRequest(CreateHttpRequestException(GetCurlException(messageResult)));
+                        completedOperation.FailRequest(CreateHttpRequestException(new CurlException(messageResult, false)));
```

Comment:
Ouch. My bad. But I believe this is the only place it got missed ? I fixed the other one.( I am assuming your comment is only about the boolean args.)